css/branding.css

```diff
@@ -0,0 +1,298 @@
+/*------------------------------------------------------------------------------
+ * Branding
+ *------------------------------------------------------------------------------
+ * @description classes associated with the branding page
+ *
+ *----------------------------------------------------------------------------*/
+
+.branding-container {
+	position: relative;
+	max-width: 100%;
+	margin-right: auto;
+	margin-left: auto;
+	padding-right: 16px;
+	padding-left: 16px;
+}
+
+@media (min-width: 576px) {
+	.branding-container {
+		width: 540px;
+	}
+}
+
+@media (min-width: 768px) {
+	.branding-container {
+		width: 720px;
+	}
+}
+
+@media (min-width: 992px) {
+	.branding-container {
+		width: 960px;
+	}
+}
+
+@media (min-width: 1200px) {
+	.branding-container {
+		width: 1140px;
+	}
+}
+
+.branding-row {
+	color: #29303d;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	margin: 16px 12px 16px -12px;
+}
+
+.branding-row-heading {
+	font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+	font-size: 36px;
```

Comment:
As stated before, I find this quite big. Maybe just something I have to get used to though.